app/src/main/java/com/aguesoftguar/medalarm/data/Medicine.java

```diff
@@ -0,0 +1,265 @@
+/*
```

Comment:
There are some indentation failures in this class.

Please, execute `Code\Reformat code` in AS